common/protocol/src/main/java/org/sdo/iotplatformsdk/common/protocol/config/SecureRandomFactory.java

```diff
@@ -18,57 +18,37 @@
 
 import java.security.NoSuchAlgorithmException;
 import java.security.SecureRandom;
+import java.util.List;
 
-import org.sdo.iotplatformsdk.common.protocol.config.SdoProperties.SecureRandomAlgorithm;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.FactoryBean;
-import org.springframework.beans.factory.FactoryBeanNotInitializedException;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
 
-@Component
-public class SecureRandomFactoryBean implements FactoryBean<SecureRandom> {
+public class SecureRandomFactory implements ObjectFactory<SecureRandom> {
 
-  private static final Logger LOG = LoggerFactory.getLogger(SecureRandomFactoryBean.class);
-  private SdoProperties properties = null;
+  private static final Logger LOG = LoggerFactory.getLogger(SecureRandomFactory.class);
+  private final List<String> secureRandomAlgorithms = List.of("NativePRNG", "NativePRNGBlocking",
+      "NativePRNGNonBlocking", "PKCS11", "SHA1PRNG", "Windows_PRNG");
```

Comment:
This list was previously defined in SdoProperties.java as a separate list, and as an enum 'SecureRandomAlgorithm'. Since the class has been removed, this list has been moved here.